cnxarchive/views.py

```diff
@@ -557,6 +561,24 @@ def search(environ, start_response):
     status = '200 OK'
     headers = [('Content-type', 'application/json')]
     start_response(status, headers)
+    
+    #In the case where a search is performed with an authorId as a filter, it is possible for the database to return no results even if the author exists in the database.  Therefore, the database is queried a second time for contact information associated with only the authorIds.  The author information is then used to update the results returned by the first database query. 
```

Comment:
We try to be [PEP8](https://www.python.org/dev/peps/pep-0008/) compliant when we can, that includes a 80char line length limit.

```
> pep8 views.py | grep 565
views.py:565:5: E265 block comment should start with '# '
views.py:565:80: E501 line too long (377 > 79 characters)
views.py:565:378: W291 trailing whitespace
```

Granted, we have more than a few long lines and other PEP8 problems in this file, but nothing over 100 chars. And when only do that when it it would otherwise make the statement look really awful.